libs/api/domains/work-machines/src/lib/workMachines.resolver.ts

```diff
@@ -129,4 +134,50 @@ export class WorkMachinesResolver {
   ) {
     return this.workMachinesService.getMachineByRegno(auth, regno, rel)
   }
+
+  @Scopes(ApiScope.vinnueftirlitid)
```

Comment:
Can this scope be in english instead?

Reply:
I think the reason that Davíð chose to have it in icelandic when he made this is because the english name is "The Administration of Occupational Safety and Health" which is quite the mouthful. But if you have a word that you think is more appropriate, I will change it.